README.md

```diff
@@ -281,6 +294,13 @@ In order to load the standard configuration file from Leiningen, add the
   See [INDENTS.md][] for a complete explanation. This will **replace**
   the default indents.
 
+* `:remove-blank-lines-in-forms?` - whether to remove blank lines inside forms
+  per the [community style
+  recommendation](https://guide.clojure.style/#no-blank-lines-within-def-forms).
```

Comment:
Use an external URL to this link, like `[community style recommendation][no-blank-lines]`.